RadixWallet/Clients/AccountLockersClient/AccountLockersClient+Live.swift

```diff
@@ -0,0 +1,210 @@
+import Foundation
+
+public typealias ClaimsPerAccount = [AccountAddress: [AccountLockerClaims]]
+
+extension AccountLockersClient {
+	public static let liveValue: Self = .live()
+
+	public static func live() -> AccountLockersClient {
+		@Dependency(\.authorizedDappsClient) var authorizedDappsClient
+		@Dependency(\.accountsClient) var accountsClient
+		@Dependency(\.onLedgerEntitiesClient) var onLedgerEntitiesClient
+		@Dependency(\.gatewayAPIClient) var gatewayAPIClient
+		@Dependency(\.cacheClient) var cacheClient
+
+		let claimsPerAccountSubject = AsyncCurrentValueSubject<ClaimsPerAccount>([:])
+
+		let timerInterval: DispatchTimeInterval = .seconds(600) // 10 minutes
```

Comment:
I think there is a static func called `minutes`?

Reply:
it isn't actually a static func, but a case of the following enum
```Swift
public enum DispatchTimeInterval : Equatable, Sendable {
    case seconds(Int)
    case milliseconds(Int)
    case microseconds(Int)
    case nanoseconds(Int)
}
```

I can make a helper static func though

Reply:
We cant use `Duration` / `Instant` types, they are from iOS 16: https://developer.apple.com/documentation/Swift/Duration

Reply:
Yes, but `AsyncTimerSequence` doesn't have an init with `Duration`, only with `DispatchTimeInterval`